node_modules/browser/browser.js

```diff
@@ -98,6 +98,8 @@ const Browser = Component(options => {
 
   const input = options.cursor('input');
 
+  const backgroundColor = webViewer.getIn(["theme", "background"]);
```

Comment:
Nit: You can actually use just `webViewer.get(["theme", "background"])` although I don't know what the diff is to be honest.